okio/src/jvmTest/kotlin/okio/internal/PriorityQueueTest.kt

```diff
@@ -20,12 +20,15 @@ import assertk.assertions.isEqualTo
 import java.lang.Integer.numberOfTrailingZeros
 import java.util.concurrent.TimeUnit
 import java.util.concurrent.TimeUnit.NANOSECONDS
+import kotlin.time.Duration.Companion.nanoseconds
+import kotlin.time.Duration.Companion.seconds
 import okio.AsyncTimeout
 import okio.PriorityQueue
 import org.junit.Test
 
 class PriorityQueueTest {
-  private val TIME_UNIT = TimeUnit.SECONDS
+  private val TIME_UNIT = TimeUnit.NANOSECONDS
+  var now = 0L
```

Comment:
We are now using timeoutAt value in node value in debug string, so set now to 0 epoch nanos to simplify debug string formatting below.

Reply:
Nice! This is a `var` but it isn’t mutated anywhere?